docs/modules/user-guide/partials/proc_adding-kubernetes-component-to-a-devfile.adoc

```diff
@@ -23,11 +23,7 @@ This section describes how to add a `kubernetes` component to a devfile. A compl
   components:
     - name: mysql
       kubernetes:
-        reference: petclinic.yaml
-        selector:
-          app.kubernetes.io/name: mysql
-          app.kubernetes.io/component: database
-          app.kubernetes.io/part-of: petclinic
+        uri: petclinic.yaml
```

Comment:
Again, cant pinpoint to the exact line in this view as they're not edited

1. `The content can be provided through the reference attribute`. 
No reference in schema, the yaml in the example seems correct

2. `Adding kubernetes component using the referenceContent attribute`
No referenceContent is schema. Both the reference and referenceContent yaml are the same

3. `Overriding container entrypoints.`
Devfile v2 does not allow this 

4. `Overriding container environment variables`
Devfile v2 allows this but not in this context or like the yaml example

5. `Specifying mount-source option`
Devfile v2 does not allow this for Kubernetes

Reply:
@jc-berger 

Adding kubernetes component to a devfile -> Adding kubernetes/openshift component to a devfile

This section describes how to add a kubernetes/openshift component to a devfile.


Prerequisites

- adding-schema-version-to-a-devfile.adoc

- adding-a-name-to-a-devfile.adoc

- adding-components-to-a-devfile.adoc

Procedure

- Define a component using the type kubernetes/openshift
- The content can be provided through `uri` or `inlined` 

   Example 1. Adding openshift component using the uri property
   ```yaml
  components:
    - name: mysql
      openshift:
        uri: petclinic.yaml
   ```

   Example 2. Adding kubernetes component using the inlined property
   ```yaml
  components:
  - name: myk8deploy
    kubernetes:
      inlined: |
        apiVersion: batch/v1
        kind: Job
        metadata:
          name: pi
        spec:
          template:
            spec:
              containers:
              - name: job
                image: myimage
                command: ["some",  "command"]
              restartPolicy: Never
   ```
- Kubernetes/Openshift components also allow you to specify endpoint throught the endpoint property
- Kubernetes/Openshift can be associated with Apply commands. If no Apply commands are associated, they are assumed to be applied at start up.